libs/clients/regulations-admin/project.json

```
@@ -17,6 +17,17 @@
         "passWithNoTests": true
       },
       "outputs": ["coverage/libs/clients/regulations-admin"]
+    },
+    "schemas/external-openapi-generator": {
```

This task should be named:
```suggestion
    "schemas/openapi-generator": {
```